src/property.ts

```diff
@@ -953,7 +948,9 @@ function Property$ensureInited(property: Property, obj: Entity): void {
 
 		// Do not initialize calculated properties. Calculated properties should be initialized using a property get rule.
 		if (!property.isCalculated) {
-			Property$init(property, obj, Property$getInitialValue(property, obj));
+			Property$init(property, obj, Property$getInitialValue(property));
+			if (property.initializer)
+				obj.update(property.name, property.initializer.call(obj));
```

Comment:
After initializing the property with its default value, use the initializer function if provided to set or update the property.